dfstools/tests/test_relationship_tools.py

```diff
@@ -32,7 +33,12 @@ def test_get_dataset_dtypes(self):
                     'trip_logs': {'flight_id': {'dtype': 'O',
                                                 # 'key_candidate': False,
                                                 'relationships': [{'flights.flight_id': {}}]}}}
-        result = find_related_cols_by_name(None, result)
+
+        dataframe_dict = {'airlines': pd.read_csv('../data/airlines/airlines.csv'),
```

Comment:
Check-out [this](https://github.com/claesmk/comp410_summer2020/blob/c0078a35fa9af3b8a4109b6f87d29de2a5a93f08/dfstools/tests/test_csv_tools.py#L27) code which gets th absolute path name of the repo you are in.  This works better than using relative backwards paths (../../..)